File: CS50x/Week6/prob/sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    while True:
        height = get_int("Height: ")
        if height >= 1 and height <= 8:
            break

    for i in range(height):
        print_row(height, i)


def print_row(height, row):
    for i in range(height - row - 1):
        print(" ", end="")
    for i in range(row + 1):
        print("#", end="")
    print("  ", end="")
    for i in range(row + 1):
        print("#", end="")
    print()


if __name__ == "__main__":
    main()

File: CS50x/Week1/problem-set/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int size;
    do
    {
        size = get_int("Height: ");
    }
    while (size < 1 || size > 8);
    for (int i = 0; i < size; i++)
    {
        for (int j = i + 1; j < size; j++)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}